x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -305,8 +310,14 @@ export async function getPackageFromSource(options: {
     if (res) {
       logger.debug(`retrieved package ${pkgName}-${pkgVersion} from cache`);
     } else {
-      res = await Registry.getPackage(pkgName, pkgVersion, { ignoreUnverified });
-      logger.debug(`retrieved package ${pkgName}-${pkgVersion} from registry`);
+      try {
```

Comment:
is there a way to check if the package is bundled before even trying EPR?

Reply:
we could change the order, I followed the same pattern as in fetchInfo [here](https://github.com/elastic/kibana/pull/147257/files#diff-fcf218bd5390498580458eeb562b2ed74a02463f3fddf280edf9545611cc57d8)

Reply:
I think it makes sense, shouldn't be a big overhead to check bundled first.

Reply:
Ah I see, I think it's something we should look into eventually but probably not now